### PR TITLE
rm 3

### PR DESCRIPTION
**Pull request by reverend-10x**
image

### CODE REVIEW

**Comment by codethazine** [comment]
Beautiful pic. Dubious contrib

File: api/server.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def do_POST(self):
             self.send_header('Access-Control-Allow-Origin', '*')
             self.end_headers()
             self.wfile.write(bytes('null', "utf-8"))
-3
```
Comment (reverend-10x) [inline api/server.py:77]: yeah i guess we should probably leave this here, huh

**Comment by codethazine** [comment]
LMAO just saw that thanks. Dunno how that ended up there

**Comment by reverend-10x** [comment]
Thank you ser @codethazine! I'm obviously new here, and just wanted to merge something trivial to get my feet wet 😅 

Perhaps my next PR could be something slightly more substantive (e.g., adding some unit tests)? Or perhaps DRY up some of the Twitter API calls like so:

image

**Pull request merged**